src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -3559,6 +3559,15 @@ paths:
                     items:
                       type: string
                     type: array
+                  service_instances:
+                    description: Service instances related to the host
+                    properties:
+                      service_instance:
+                        description: Number of service instances
+                        type: integer
+                    required:
+                    - service_instance
+                    type: object
```

Comment:
That's still not correct. That schema will fail against anything other than:
```json
{
  "service_instance": 2
}
```

However, we are expecting:
```json
{
  "mon": 3,
  "mgr": 1
}
```

If we struggle to define this, then it's better (yet not best practice) to leave it undefined (no `properties` or `required`).